packages/webpack/src/unplugin.ts

```diff
@@ -157,7 +157,7 @@ export function unplugin<Theme extends object>(configOrPath?: WebpackPluginOptio
       lastTokenSize = tokens.size
       Array.from(plugin.__vfsModules)
         .forEach((id) => {
-          let path = decodeURIComponent(id.slice(plugin.__virtualModulePrefix.length))
+          let path = decodeURIComponent(id.replace(plugin.__virtualModulePrefix, ''))
```

Comment:
I wonder if we should do `id.startsWith(plugin.__virtualModulePrefix) ? id.slice(plugin.__virtualModulePrefix.length) : id`

Reply:
Yes, it is better. I updated